tests/lib/managers/webhooks-test.js

```diff
@@ -0,0 +1,158 @@
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
+			sandbox.stub(boxClientFake, 'post').withArgs('/webhooks').yieldsAsync();
+			webhooks.create(ID, TYPE, ADDRESS, TRIGGERS, done);
+		});
+	});
+
+	describe('get()', function() {
+
+		it('should make GET request to get Webhook info when called', function() {
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('get').withArgs('/webhooks/1234', testParamsWithQs);
+			webhooks.get(WEBHOOKS_ID, testQS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
+			sandbox.stub(boxClientFake, 'get').withArgs('/webhooks/1234', testParamsWithQs).yieldsAsync();
+			webhooks.get(WEBHOOKS_ID, testQS, done);
+		});
+	});
+
+	describe('getAll()', function() {
+
+		it('should make GET call to fetch all webhooks', function() {
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('get').withArgs('/webhooks', testParamsWithQs);
+			webhooks.getAll(testQS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
+			sandbox.stub(boxClientFake, 'get').withArgs(testParamsWithQs).yieldsAsync();
+			webhooks.getAll(testQS, done);
+			done();
+		});
+	});
+
+	describe('update()', function() {
+
+		var param = {
+			target: {
+				id: '1234',
+				type: 'file'
+			},
+			address: 'https://www.test1.com',
+			triggers: ['FILE.DOWNLOADED', 'FILE.PREVIEWED']
+		};
+		it('should make PUT call to update webhook', function() {
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('put').withArgs('/webhooks/1234');
+			webhooks.update(WEBHOOKS_ID, param);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
+			sandbox.stub(boxClientFake, 'put').withArgs('/webhooks/1234').yieldsAsync();
+			webhooks.update(WEBHOOKS_ID, param, done);
+		});
+	});
+
+	describe('delete()', function() {
+
+		it('should make DELETE call to remove webhook', function() {
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('del').withArgs('/webhooks/1234');
+			webhooks.delete(WEBHOOKS_ID, testQS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
+			sandbox.stub(boxClientFake, 'del').withArgs('/webhooks/1234').yieldsAsync();
+			webhooks.delete(WEBHOOKS_ID, testQS, done);
```

Comment:
This is why you need to assert about the arguments to defaultResponseHandler!  This test as written should fail, because your `delete()` implementation does not include a second `qs` parameter and you're passing one here.  If you called the function like this in reality, you'd get an error because the SDK would try to call testQS as a callback